src/python/pants/backend/python/tasks/pytest_run.py

```diff
@@ -109,7 +110,7 @@ def register_options(cls, register):
                   "emitted to that file (prefix). Note that tests may run in a different cwd, so "
                   "it's best to use an absolute path to make it easy to find the subprocess "
                   "profiles later.")
-
+    register('--options', type=list, fingerprint=True, help='Pass these options to pytest.')
```

Comment:
Perhaps helpful to point out that `passthrough` args are supported for this option.